apps/server/src/services/project-service/ProjectService.ts

```
@@ -296,13 +304,22 @@ export async function patchCurrentProject(data: Partial<DatabaseModel>) {
+    const customFields = parseCustomFields(data);
+    const result = parseRundowns(data, customFields);
+
     /**
      * The user may have multiple rundowns
      * We currently ignore all other rundowns
      */
-    const firstRundown = getFirstRundown(result.rundowns);
-    await initRundown(firstRundown, result.customFields);
+    const firstRundown = getFirstRundown(result);
+    // eslint-disable-next-line no-unused-labels -- dev code path
+    DEV: {
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)